src/generate_supplemental_weekly_plot.R

```diff
@@ -8,14 +8,36 @@ generate_supplemental_weekly_plot <- function(daily_plot_data_arg, df_list_arg,
   week_plot_data <- site_param_df %>%
     filter(DT_round %within% interval(start_date, end_date))
 
-  site_vector <- c("tamasag",
-                    "legacy",
-                    "lincoln",
-                    "timberline",
-                    "prospect",
-                    "boxelder",
-                    "archery",
-                    "river bluffs")
+  #default is lower network
+  site_vector <- c("tamasag", # rist
+                   "legacy",
+                   "lincoln",
+                   "timberline",
+                   "prospect",
+                   "boxelder", # elc
+                   "archery",
+                   "river bluffs")
+
+  if(network == "virridy"){
+    site_vector <-  c("joei",
+                      "cbri",
+                      "chd",
+                      "pfal",
+                      "pbd",
+                      "sfm",
+                      "lbea",
+                      "penn",
+                      NA,
+                      "lincoln",
+                      "timberline",
+                      "timberline virridy",
+                      "springcreek",
+                      "prospect",
+                      "prospect virridy",
+                      "archery",
+                      "archery virridy",
+                      "boxcreek")
+  }
```

Comment:
To do: The upper and lower lines for the plots generated in this function are based on the indexes of this list. Based on this list I don't think the upper and lower plots will always be displayed (though I have not tried running this yet). Example: If you are verifying `timberline` the upper site plotted will be `lincoln`, but the lower site will be `timberline virridy` instead of `springcreek`. If this is expected/fine then it is fine to ignore this. Tagging this in case that is unexpected behavior.

Reply:
It's my stop-gap measure for now! Definitely need to update. Thanks for flagging.